### PR TITLE
Blog post: 2022 in review and adding social media card preview

### PR DESCRIPTION
**Pull request by choldgraf**
This PR does two things:

### Blog post

Adds a blog post for our 2022 year in review. This covers some major highlights that I recalled from the last year and a few ideas for where to prioritize our time in the next one!

**[Here's a link to the post](https://deploy-preview-130--cocky-booth-e7ed17.netlify.app/blog/2023/2022-year-in-review/)**

### Social media card templates

Adds some Hugo infrastructure to generate social media previews, inspired by @jacobtomlinson's awesome implementation [here](https://github.com/jacobtomlinson/website/blob/6028b0d95a09f82de84b8c7f0e8b568be0bf879f/themes/working/layouts/partials/ogimage.html). This means we don't have to hand-add any more social media previews :-)

Here's a preview of what the twitter card preview will look like:

image

This also adds some minor style tweaks to make the headers behave more similarly to our documentation.

### CODE REVIEW

**Comment by choldgraf** [comment]
Took another pass and made a few more improvements. **I will publish this later today if nobody suggests otherwise**.

File: content/blog/2023/2022-year-in-review/index.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+This marks our first efforts into _development-focused work_ as opposed to operating cloud infrastructure.
+We will use this experience to learn how to pair focused development with cloud operations (more on that below).
+It will also make it more likely that we can implement often-requested improvements to the JupyterHub / BinderHub ecosystems.
+[Here's a blog post about this collaboration](https://2i2c.org/blog/2022/gesis-2i2c-collaboration-update/).
+
+### We helped maintain several upstream open source projects
+
+We made a number of contributions to key open source communities as part of our organizational mission.
+These spanned technical improvements as well as organizational and community efforts.
+One highlight is that several team members have participated in [JupyterHub's latest round of Outreachy interns](https://blog.jupyter.org/introducing-jupyterhubs-outreachy-interns-december-2022-cohort-23aaf4613556?source=collection_home---6------4-----------------------).
```
Comment (damianavila) [inline content/blog/2023/2022-year-in-review/index.md]:
```suggestion
One highlight is that several team members have participated in [JupyterHub's latest round of Outreachy interns](https://blog.jupyter.org/introducing-jupyterhubs-outreachy-interns-december-2022-cohort-23aaf4613556).  
```
That long url feels like a tracking one...

**Review by damianavila — APPROVED** [review]
A minor request to update one URL, otherwise looks great to me.
Thanks for writing it!!

**Comment by choldgraf** [comment]
ok merged - thanks @damianavila for taking a look!

**Pull request merged**